src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -83,6 +83,7 @@ def load_arguments(self, _):
         c.argument('pod_cidr')
         c.argument('service_cidr')
         c.argument('vnet_subnet_id', type=str, validator=validate_vnet_subnet_id)
+        c.argument('ppg')
```

Comment:
Do you need to add some validation fo the ppg string?

Reply:
There is validation in rp sync code.